nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -214,6 +215,22 @@
             .addValidator(StandardValidators.TIME_PERIOD_VALIDATOR)
             .build();
 
+    public static final PropertyDescriptor PROP_IDLE_TIMEOUT = new PropertyDescriptor.Builder()
+            .name("Idle Timeout")
```

Comment:
I missed this. But can you change this to `.displayName("Idle Timeout")` and add `.name("idle-timeout")`? Similar for the other property. I know the older props didn't do this, but it's good to do for newer ones. If you can't get to it, I can maybe do it on merge since it's a tiny change.